wetmap/src/components/googleMap.jsx

```diff
@@ -477,13 +477,13 @@ function Map() {
   };
 
   const setupShopModal = async (shopName) => {
-    modalShow(ShopModal, {
-      size: ModalWindowSize.L,
-    });
     setSelectedShop({ id: 0, orgName: 'hello' });
     let chosenShop = await getShopByName(shopName);
     setSelectedShop(chosenShop);
-    setShopModal(true);
+    modalShow(ShopModal, {
+      size: ModalWindowSize.L,
+    });
+    // setShopModal(true);
```

Comment:
what was the purpose of commenting out this line, I'm assuming that the setShopModal and the modalShow methods have the same functionality, but are there any specific differences with how the shopmodal context works compared to the modalShow functionality?